app/views/conferences/_details.html.slim

```diff
@@ -6,7 +6,8 @@
     dd = @conference.date_range
     dt Twitter
     dd = format_conference_twitter(@conference.twitter)
-    dt Homepage
-    dd = link_to @conference.url, @conference.url
+    -if @conference.url
```

Comment:
Cool! Thanks Team! ✨ 
I'd vote for leaving the label in, even when the url value is empty. Same as we do with the other labels.